cpp/zotero_harvester.cc

```diff
@@ -178,9 +188,12 @@ struct HarvesterConfigData {
 };
 
 
-void LoadHarvesterConfig(const std::string &config_path, HarvesterConfigData * const harvester_config) {
+void LoadHarvesterConfig(const std::string &config_path, HarvesterConfigData * const harvester_config,
+                         const IniFile::Section &config_overrides=IniFile::Section())
+{
     Config::LoadHarvesterConfigFile(config_path, &harvester_config->global_params_,
-                                    &harvester_config->group_params_, &harvester_config->journal_params_);
+                                    &harvester_config->group_params_, &harvester_config->journal_params_,
+                                    nullptr, config_overrides);
```

Comment:
Please insert /* dummy_argument_name = */ in front of nullptr, w/ the appropriate dummy argument name.